questions.md

```diff
@@ -0,0 +1,3 @@
+# Questions
+
+1. Do you recommend using GitHub Desktop GUI for use in class with work groups for branches, merges and pull requests?
```

Comment:
I do not for absolute beginners--I would recommend using the CLI for pulling to have students in the habit of it. Also would recommend this tool for students as they begin to work on team projects: https://git-fork.com/